components/reports/temperature/TempReport.vue

```diff
@@ -61,11 +64,5 @@ export default {
       temp_season: 'DJF',
     }
   },
-  computed: {
```

Comment:
Oh this wasn't needed because the actual data lives in the sub-reports huh?  Good catch!